src/context/viewer-context.tsx

```diff
@@ -56,6 +56,15 @@ function useAuthedViewer() {
     const loadViewerFromStorage = async () => {
       console.log(`loading viewer from storage`)
 
+      const localViewer = auth.getLocalUser()
+
+      if (localViewer) {
+        if (!isEqual(localViewer.id, viewerId)) {
+          setViewer(localViewer)
+        }
+        setLoading(() => false)
```

Comment:
this was causing a delay, but if we have a viewer in localstorage might as well call it loaded optimistically and use that while a refresh happens in the background